api-reference/10 UI Widgets/GridBase/1 Configuration/editing/confirmDelete.md

```diff
@@ -5,7 +5,7 @@ default: true
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies if the confirmation is required when you delete a row.
```

Comment:
```suggestion
Specifies if the confirmation is required when a user deletes a row.
```

Reply:
We also need to edit this description: https://js.devexpress.com/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/editing/texts/#confirmDeleteMessage